lib/checks/aria/aria-hidden-focus.js

```diff
@@ -0,0 +1,27 @@
+const domTree = axe.utils.querySelectorAll(virtualNode, '*');
+
+const relatedNodes = [];
+
+const result = domTree.every(({ actualNode: el }) => {
+	const isElFocusable = axe.commons.dom.isFocusable(el);
+	let tabIndex = el.getAttribute('tabindex');
+	// dom.isFocusable does not check for tabindex
```

Comment:
`dom.isFocusable` doesn't check for tabindex? Are you referring to a newer version that hasn't been merged yet? Because I'm looking at tabindex logic in that file. Tabindex impacts focusability, so it definitely should be included.